### PR TITLE
Treat emoji characters differently 

### PR DESCRIPTION
**Pull request by heytitle**
REF: #361

### CODE REVIEW

File: lib/DDG/Goodie/UnicodeFuzzySearch.pm [hunk 1]
```diff
@@ -2,6 +2,7 @@ package DDG::Goodie::UnicodeFuzzySearch;
 # ABSTRACT: returns unicode symbols matching the input
 
 use DDG::Goodie;
+use URI::Escape::XS;
 
 triggers startend => "unicode";
```
Comment (jagtalon) [inline lib/DDG/Goodie/UnicodeFuzzySearch.pm]: @heytitle That is weird! Try just '/iu/?u='--that worked for me.

screen shot 2015-01-05 at 4 04 37 pm

**Comment by heytitle** [comment]
@jagtalon @moollaza  I'm not sure which emoji provider we should use between `emoji-sheet-cheat` or `Twitter ( twemoji )`. In my opinion, Twitter seems to a good choice. However its icons are totally different from normal, so it could confuse users a bit.

**Comment by jagtalon** [comment]
:+1:

**Comment by heytitle** [comment]
Now, the epic is ended :fireworks:

**Comment by mintsoft** [comment]
Congrats @heytitle !

**Comment by jagtalon** [comment]
Yeah, @heytitle congrats! :D

**Comment by jagtalon** [comment]
@heytitle live! https://duckduckgo.com/?q=emoji+cyclone&ia=answer

**Comment by jkanarek** [comment]
Broken in a couple places: https://duckduckgo.com/?q=emoji+poo&ia=answer

**Comment by jagtalon** [comment]
@jkanarek ah nice catch! CC @heytitle

**Comment by jkanarek** [comment]
Why do the non-emoji characters appear on that query BTW?

**Comment by RomainLG** [comment]
Is this feature broken?

I don't seem to be able to get emojis to work:

https://duckduckgo.com/?q=emoji+cyclone&ia=answer

**Comment by heytitle** [comment]
Look like Chrome supports Emoji now.
cc: @jagtalon @moollaza

**Comment by moollaza** [comment]
>Is this feature broken?

>I don't seem to be able to get emojis to work:

@RomainLG this IA was taken down because of the lack of Emoji support across browser. It led to some bad looking results.

>Look like Chrome supports Emoji now.
http://blog.getemoji.com/post/105296903020/google-chrome-finally-supports-emoji-and-its
@heytitle glad to hear Chrome is finally gaining support.

**Pull request merged**